frame/vamm/src/tests/swap.rs

```diff
@@ -440,19 +459,19 @@ proptest! {
 
 		swap_config.vamm_id = VammId::zero();
 
-		ExtBuilder {
-			vamm_count: 1,
-			vamms: vec![(0, vamm_state)]
-		}.build().execute_with(|| {
-			// For event emission.
-			run_for_seconds(vamm_state.twap_period);
-			if TestPallet::swap(&swap_config).is_ok() {
-				let new_vamm_state = TestPallet::get_vamm(0).unwrap();
-				System::assert_has_event(
-					Event::UpdatedTwap { vamm_id: 0, base_twap: new_vamm_state.base_asset_twap }.into(),
-				);
+		with_existent_vamm_swap_contex(
+			vamm_state,
+			swap_config,
+			|swap_config| {
+				// For event emission.
+				run_for_seconds(vamm_state.twap_period);
```

Comment:
Could perhaps try with several time periods instead of just `twap_period`, e.g., after `1..=X` seconds

Reply:
Done :+1: 
Thanks for the suggestion! :)